.ci/deps.sh

```diff
@@ -175,3 +175,12 @@ fi
 echo '' > dummy
 coala-ci --bears CheckstyleBear --files dummy --no-config --bear-dirs bears || true
 coala-ci --bears ScalaLintBear --files dummy --no-config --bear-dirs bears || true
+
+if [ ! -e ~/elm_format/elm-format ]; then
+  mkdir -p ~/elm_format
+  export PATH=$PATH:~/elm_format
+  wget -nc -O elm-format.tgz https://github.com/avh4/elm-format/releases/download/0.5.2-alpha/elm-format-0.17-0.5.2-alpha-linux-x64.tgz
```

Comment:
I'd prefer we start using curl only, as `wget` isnt installed on the docker (whereas curl is mandatory in the docker), and it would be good for the docker to be directly using the coala-bears install scripts, rather than having a slightly different set of commands.  see https://github.com/coala/docker-coala-base/blob/master/Dockerfile